common/metrics/metricstest/metricstest.go

```diff
@@ -102,6 +134,7 @@ func (h *Handler) Snapshot() (Snapshot, error) {
 		return Snapshot{}, err
 	}
 	samples := map[string]sample{}
+	histogramSamples := map[string]histogramSample{}
 	for name, family := range families {
 		for _, m := range family.GetMetric() {
```

Comment:
Could you extract this for loop to a function? It's starting to get a little large